apps/mitt-konto/src/MittKonto/Main/Views/LoginView.purs

```diff
@@ -0,0 +1,99 @@
+module MittKonto.Main.Views.LoginView where
```

Comment:
I would prefer these without the "View" suffix. They're all views in that directory.

Reply:
🤔  I have to confess I prefer having the module name mirroring the path of the file in general, I think it's clearer. If I was forced to name one positive aspect of Java, that's what I would mention 😆 

But I see your point! Let's change it